api/opencl_runtime_layer.asciidoc

```diff
@@ -4344,8 +4385,27 @@ include::{generated}/api/version-notes/CL_IMAGE_D3D11_SUBRESOURCE_KHR.asciidoc[]
       specified when _image_ was created.
 endif::cl_khr_d3d11_sharing[]
 
+ifdef::cl_ext_image_tiling_control[]
+| {CL_IMAGE_TILING_EXT_anchor}
+
+include::{generated}/api/version-notes/CL_IMAGE_TILING_EXT.asciidoc[]
+  | {cl_image_tiling_ext_TYPE}
+    | Return the tiling passed using the {CL_MEM_IMAGE_TILING_EXT} property or
+      {CL_IMAGE_TILING_LINEAR_EXT} if none was passed at image creation time.
```

Comment:
Discussed in the 2025/01/28 memory TSG: agreement to return the tiling that was either specified by the application at image creation time or selected by the implementation in the absence of tiling specified by the application.

Reply:
I have now reflected the agreed change in the draft specification.